### ISSUE TITLE
lldp start-time='3 minutes ago' prodces exception: Error running command: Can only use .dt accessor with datetimelike values

### ISSUE THREAD

**Issue by jopietsch** [netenglabs/suzieq#300]
**Describe the bug**
```
jpietsch> lldp show start-time='3 minutes ago'
Error running command: Can only use .dt accessor with datetimelike values
------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jpietsch/.cache/pypoetry/virtualenvs/suzieq-QZqWZG2D-py3.8/src/python-nubia/nubia/internal/cmdbase.py", line 408, in run_interactive
    ret = fn(**args_dict)
  File "/home/jpietsch/suzieq/suzieq/cli/sqcmds/LldpCmd.py", line 49, in show
    df = self.sqobj.get(
  File "/home/jpietsch/suzieq/suzieq/sqobjects/basicobj.py", line 155, in get
    return self.engine_obj.get(**kwargs)
  File "/home/jpietsch/suzieq/suzieq/engines/pandas/engineobj.py", line 144, in get
    df = self.get_valid_df(self.iobj.table, **kwargs)
  File "/home/jpietsch/suzieq/suzieq/engines/pandas/engineobj.py", line 132, in get_valid_df
    table_df['timestamp'] = humanize_timestamp(table_df.timestamp)
  File "/home/jpietsch/suzieq/suzieq/utils.py", line 554, in humanize_timestamp
    return field.apply(lambda x: datetime.fromtimestamp(x/1000)) \
  File "/home/jpietsch/.cache/pypoetry/virtualenvs/suzieq-QZqWZG2D-py3.8/lib/python3.8/site-packages/pandas/core/generic.py", line 5135, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/jpietsch/.cache/pypoetry/virtualenvs/suzieq-QZqWZG2D-py3.8/lib/python3.8/site-packages/pandas/core/accessor.py", line 187, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/jpietsch/.cache/pypoetry/virtualenvs/suzieq-QZqWZG2D-py3.8/lib/python3.8/site-packages/pandas/core/indexes/accessors.py", line 480, in __new__
    raise AttributeError("Can only use .dt accessor with datetimelike values")
AttributeError: Can only use .dt accessor with datetimelike values
```

**Comment by jopietsch** [comment]
interface has the same problem:
```
(suzieq-QZqWZG2D-py3.8) jpietsch@jpietsch-server:~/suzieq$ suzieq/cli/suzieq-cli interface show --start-time='one hour ago'
Error running command: Can only use .dt accessor with datetimelike values
------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jpietsch/.cache/pypoetry/virtualenvs/suzieq-QZqWZG2D-py3.8/src/python-nubia/nubia/internal/cmdbase.py", line 482, in run_cli
    return fn(**kwargs)
  File "/home/jpietsch/suzieq/suzieq/cli/sqcmds/InterfaceCmd.py", line 56, in show
    df = self.sqobj.get(
  File "/home/jpietsch/suzieq/suzieq/sqobjects/basicobj.py", line 155, in get
    return self.engine_obj.get(**kwargs)
  File "/home/jpietsch/suzieq/suzieq/engines/pandas/engineobj.py", line 144, in get
    df = self.get_valid_df(self.iobj.table, **kwargs)
  File "/home/jpietsch/suzieq/suzieq/engines/pandas/engineobj.py", line 132, in get_valid_df
    table_df['timestamp'] = humanize_timestamp(table_df.timestamp)
  File "/home/jpietsch/suzieq/suzieq/utils.py", line 554, in humanize_timestamp
    return field.apply(lambda x: datetime.fromtimestamp(x/1000)) \
  File "/home/jpietsch/.cache/pypoetry/virtualenvs/suzieq-QZqWZG2D-py3.8/lib/python3.8/site-packages/pandas/core/generic.py", line 5135, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/jpietsch/.cache/pypoetry/virtualenvs/suzieq-QZqWZG2D-py3.8/lib/python3.8/site-packages/pandas/core/accessor.py", line 187, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/jpietsch/.cache/pypoetry/virtualenvs/suzieq-QZqWZG2D-py3.8/lib/python3.8/site-packages/pandas/core/indexes/accessors.py", line 480, in __new__
    raise AttributeError("Can only use .dt accessor with datetimelike values")
AttributeError: Can only use .dt accessor with datetimelike values
------------------------------------------------------------
```

**Comment by jopietsch** [comment]
address, arpnd, interfaces, lldp are broken

bgp, device, fs, macs, mlag, ospf, routes, sqPoller, tables, vlan work

**Comment by ddutt** [comment]
Fixed with commit 3ee103759b055e947b274033b33939931716a0bb
